app/controllers/trainees/confirm_withdrawals_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Trainees
+  class ConfirmWithdrawalsController < ApplicationController
+    def show
+      authorize trainee
+    end
+
+    def update
+      authorize trainee
+      redirect_to edit_trainee_path(trainee)
+    end
+
+  private
+
+    def trainee
+      @trainee ||= Trainee.find(params[:trainee_id])
+    end
+
+    def trainee_params
```

Comment:
Is the `trainee_params` redundant? I don't see it used anywhere here

Reply:
Yes. Good spot.